### PR TITLE
Translated create-a-new-react-app.md

### PR DESCRIPTION
**Pull request by JohannesKlauss**
Several words I am not sure how to translate properly:

- Frontend Build Pipeline
- out of the box (it's actually in the duden but as a EDV term - https://www.duden.de/rechtschreibung/out_of_the_box)
- Package Manager (here translated to Paket Manager)
- third-party packages (here translated to Drittbibliotheken, since Drittpakete doesn't make any sense in german)

I feel like translating **toolchain** to **Ansammlung von Werkzeugen** is super confusing for a german reader, since it looses the meaning of a **chain**. I translated it to **Werkzeuge** or **Werkzeugkette** (depending on the context), but we can certainly discuss this further.

### CODE REVIEW

**Comment by ph1p** [comment]
> Several words I am not sure how to translate properly:
> 
> * Frontend Build Pipeline
> * out of the box (it's actually in the duden but as a EDV term - https://www.duden.de/rechtschreibung/out_of_the_box)
> * Package Manager (here translated to Paket Manager)
> * third-party packages (here translated to Drittbibliotheken, since Drittpakete doesn't make any sense in german)
> 
> I feel like translating **toolchain** to **Ansammlung von Werkzeugen** is super confusing for a german reader, since it looses the meaning of a **chain**. I translated it to **Werkzeuge** or **Werkzeugkette** (depending on the context), but we can certainly discuss this further.

I think as a devs we know all of these words and if not, we have the glossary page. 

I added all of these to the glossary issue (#10)

* Frontend Build-Pipeline
* Out of the box -> Some sort of "Buzzword" 
* I think Package Manager and Paket Manager is also ok
* Pakete von Drittanbietern -> Maybe something like this
* Werkzeugkette -> Toolchain is a really tricky word. I still don't know the best solution.

File: content/docs/create-a-new-react-app.md [hunk 1]
```diff
@@ -1,94 +1,94 @@
...
-Learn Next.js from [its official guide](https://nextjs.org/learn/).
+Lerne Next.js durch den [offizielen Leitfaden](https://nextjs.org/learn/).
 
 ### Gatsby {#gatsby}
 
-[Gatsby](https://www.gatsbyjs.org/) is the best way to create **static websites** with React. It lets you use React components, but outputs pre-rendered HTML and CSS to guarantee the fastest load time.
+[Gatsby](https://www.gatsbyjs.org/) ist der beste Weg, um mit React **statische Webseiten** zu erstellen. Es lässt dich React Komponenten benutzen, erzeugt aber vorgerendertes HTML und CSS, um die schnellstmögliche Ladezeit zu garantieren.
 
-Learn Gatsby from [its official guide](https://www.gatsbyjs.org/docs/) and a [gallery of starter kits](https://www.gatsbyjs.org/docs/gatsby-starters/).
+Lerne Gatsby durch den [offiziellen Leitfaden](https://www.gatsbyjs.org/docs/) und einer [Gallerie mit Starter Kits](https://www.gatsbyjs.org/docs/gatsby-starters/).
```
Comment (ph1p) [inline content/docs/create-a-new-react-app.md]:
```suggestion
Lerne Gatsby durch den [offiziellen Leitfaden](https://www.gatsbyjs.org/docs/) und einer [Galerie mit Starter Kits](https://www.gatsbyjs.org/docs/gatsby-starters/).
```

File: content/docs/create-a-new-react-app.md [hunk 2]
```diff
@@ -1,94 +1,94 @@
...
+Wenn du bereit bist auf die Produktionsumgebung zu deployen, führe `npm run build` aus. Das produziert einen optimierten Build für deine Anwendung im `build` Ordner. Du kannst mehr über Create React App [in dessen README](https://github.com/facebookincubator/create-react-app#create-react-app--) lernen und im [Benutzer Leitfaden](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents).
 
 ### Next.js {#nextjs}
 
-[Next.js](https://nextjs.org/) is a popular and lightweight framework for **static and server‑rendered applications** built with React. It includes **styling and routing solutions** out of the box, and assumes that you're using [Node.js](https://nodejs.org/) as the server environment.
+[Next.js](https://nextjs.org/) ist ein beliebtes und leichtes Framework für mit React gebaute, **statische und vom Server gerenderte Anwendungen**. Es beinhaltet **Styling- und Routinglösungen** out of the box und nimmt an, dass du als Serverumgebung [Node.js](https://nodejs.org/) verwendest.
```
Comment (ph1p) [inline content/docs/create-a-new-react-app.md]:
```suggestion
[Next.js](https://nextjs.org/) ist ein beliebtes und leichtgewichtiges Framework für mit React gebaute, **statische und vom Server gerenderte Anwendungen**. Es beinhaltet **Styling- und Routinglösungen** out of the box und nimmt an, dass du als Serverumgebung [Node.js](https://nodejs.org/) verwendest.
```

File: content/docs/create-a-new-react-app.md [hunk 3]
```diff
@@ -1,94 +1,94 @@
...
+Create React App handhabt weder Backend-Logik noch Datenbanken; es erzeugt einfach nur eine Frontend Build Pipeline, so dass du es mit jedem gewünschten Backend verwenden kannst. Unter der Haube verwendet es [Babel](https://babeljs.io/) und [webpack](https://webpack.js.org/), aber darüber musst du nichts wissen.
 
-When you're ready to deploy to production, running `npm run build` will create an optimized build of your app in the `build` folder. You can learn more about Create React App [from its README](https://github.com/facebookincubator/create-react-app#create-react-app-) and the [User Guide](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents).
+Wenn du bereit bist auf die Produktionsumgebung zu deployen, führe `npm run build` aus. Das produziert einen optimierten Build für deine Anwendung im `build` Ordner. Du kannst mehr über Create React App [in dessen README](https://github.com/facebookincubator/create-react-app#create-react-app--) lernen und im [Benutzer Leitfaden](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents).
```
Comment (ph1p) [inline content/docs/create-a-new-react-app.md]:
```suggestion
Wenn du bereit bist auf die Produktionsumgebung zu deployen, führe `npm run build` aus. Dies erstellt einen optimierten Build für deine Anwendung im `build` Ordner. Du kannst mehr über Create React App [in der README](https://github.com/facebookincubator/create-react-app#create-react-app--) und dem [Benutzerhandbuch](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents) erfahren.
```

File: content/docs/create-a-new-react-app.md [hunk 4]
```diff
@@ -1,94 +1,94 @@
...
+- Wenn du **React lernst** oder eine neue [Single Page](/docs/glossary.html#single-page-application) Anwendung erstellst, dann benutze [Create React App](#create-react-app).
+- Wenn du eine **vom Server gerenderte Webseite mit Node.js** baust, dann probier [Next.js](#nextjs).
+- Wenn du eine **statische auf Inhalt orientierte Webseite baust**, dann probier [Gatsby](#Gatsby).
+- Wenn du eine **Komponentenbibliothek** baust oder **mit einer existierenden Codebasis integrierst**, dann probiere [flexiblere Werkzeuge](#more-flexible-toolchains).
 
 ### Create React App {#create-react-app}
 
-[Create React App](https://github.com/facebookincubator/create-react-app) is a comfortable environment for **learning React**, and is the best way to start building **a new [single-page](/docs/glossary.html#single-page-application) application** in React.
+[Create React App](https://github.com/facebookincubator/create-react-app) ist eine komfortable Umgebung, um **React zu lernen** und ist der beste Weg, mit einer neuen [Single Page](/docs/glossary.html#single-page-application) Anwendung in React zu beginnen.
```
Comment (ph1p) [inline content/docs/create-a-new-react-app.md]:
```suggestion
[Create React App](https://github.com/facebookincubator/create-react-app) ist eine komfortable Umgebung, um **React zu lernen** und ist der beste Weg, mit **einer neuen [Single Page](/docs/glossary.html#single-page-application) Anwendung** in React zu beginnen.
```

File: content/docs/create-a-new-react-app.md [hunk 5]
```diff
@@ -1,94 +1,94 @@
...
+## Empfohlene Werkzeuge {#recommended-toolchains}
 
-The React team primarily recommends these solutions:
+Das React Team empfiehlt bevorzugt diese Lösungen:
 
-- If you're **learning React** or **creating a new [single-page](/docs/glossary.html#single-page-application) app,** use [Create React App](#create-react-app).
-- If you're building a **server-rendered website with Node.js,** try [Next.js](#nextjs).
-- If you're building a **static content-oriented website,** try [Gatsby](#gatsby).
-- If you're building a **component library** or **integrating with an existing codebase**, try [More Flexible Toolchains](#more-flexible-toolchains).
+- Wenn du **React lernst** oder eine neue [Single Page](/docs/glossary.html#single-page-application) Anwendung erstellst, dann benutze [Create React App](#create-react-app).
```
Comment (ph1p) [inline content/docs/create-a-new-react-app.md]:
```suggestion
- Wenn du **React lernst** oder **eine neue [Single Page](/docs/glossary.html#single-page-application) Anwendung** erstellst, dann benutze [Create React App](#create-react-app).
```

**Comment by JohannesKlauss** [comment]
@ph1p I committed all your suggestions. Thanks.

**Comment by ph1p** [comment]
> @ph1p I committed all your suggestions. Thanks.

You're welcome. I will merge this as first version. Maybe with more and more pages we get a feeling for some of these words.

**Pull request merged**